bridgetown-website/src/_docs/content/pagination.md

```diff
@@ -40,7 +40,7 @@ Then you can use the `paginator.resources` logic to iterate through the collecti
 
 {%@ Documentation::Multilang do %}
 ```erb
-<% paginator.each do |post| %>
+<% paginator.resources.each do |post| %>
```

Comment:
Let's revert this change, I'm convinced we don't need to keep that `resources` method call and if we need to fix a bug to get it working we will (provided I can reproduce).